### ISSUE TITLE
S3512 should not report on 2 operands

### ISSUE THREAD

**Issue by vilchik-elena** [SonarSource/SonarJS#1943]
```
let x = "Hello " + name; // reporting in such cases makes rule noisy
```
